src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -651,21 +651,28 @@ function disableRow(elem, row) {
     $tdElements.parent().hide();
 }
 
-function feedbackRecipientUpdateVisibilityOptions(elem) {
-    elem = $(elem);
-    if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
-        enableRow(elem, 1);
-        disableRow(elem, 3);
+function feedbackRecipientUpdateVisibilityOptions(recipientType) {
+    $recipientType = $(recipientType);
+    if (isRecipientsTeamMembersVisibilityOptionValidForRecipientType($recipientType.val())) {
+        // show the row Recipient(s) and hide the row Recipient's Team Members
```

Comment:
@kanghj I rename the method as isRecipientsTeamMembersVisibilityOptionHiddenForRecipientType. Is that better?